tensorflow/python/eager/def_function.py

```diff
@@ -456,7 +456,8 @@ def __init__(self,
                experimental_implements=None,
                experimental_autograph_options=None,
                experimental_relax_shapes=False,
-               experimental_compile=None):
+               experimental_compile=None,
+               experimental_type_tracing=False):
```

Comment:
Let's name this `experimental_follow_type_hints` for now. We want to avoid names which are too specific and keep the API simple.

Reply:
Sounds good, I changed the name to `experimental_follow_type_hints`.